app.go

```diff
@@ -1,56 +1,36 @@
 package main
 
 import (
+	"flag"
 	"fmt"
 	"log"
 	"os"
 	"path"
 	"qurandw/api"
 	"qurandw/utils"
 	"slices"
-	"strconv"
 	"sync"
 )
 
 func main() {
-	locale := "ar"
-	concurrentDownloads := 10
-	reverseOrder := false
-	outputDir, err := os.Getwd()
+	locale := flag.String("locale", "ar", "--locale <language>")
+	concurrentDownloads := flag.Int("batches", 10, "--batches <amount>")
+	reverseOrder := flag.Bool("reverse", false, "--reverse <true/flase>")
```

Comment:
typo `true/flase`

Reply:
Thank you for pointing this out, I've fixed it in my latest commit.